test/apis/pytorch/text-generator/requirements.txt

```diff
@@ -1,2 +1,2 @@
-torch
+torch==1.7.*
```

Comment:
What do you think of doing this for all the other `requirements.txt` for all of our examples?

Reply:
Since we plan on reverting this change once 1.8 becomes more stable, I think it is okay if we do enough to get the tests passing.

Reply:
Yeah, that sounds good.